tracetool/main.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+###
+### Copyright (C) 2021 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
```

Comment:
libva-utils is licensed under MIT, what's the intent to introduce another one?

Reply:
updated to MIT license header